src/game/client/components/nameplates.cpp

```diff
@@ -109,6 +111,37 @@ void CNamePlates::RenderNameplate(
 			}
 		}
 
+		if(g_Config.m_ClAfkMark && m_pClient->m_aClients[ClientID].m_Afk)
+		{
+			const char *afkMark = "!";
+			if(str_comp(afkMark, m_aNamePlates[ClientID].m_aAfkMark) != 0 || (FontSize - FontOffsetAfkMark) != m_aNamePlates[ClientID].m_AfkMarkFontSize)
+			{
+				mem_copy(m_aNamePlates[ClientID].m_aAfkMark, afkMark, sizeof(m_aNamePlates[ClientID].m_aAfkMark));
+				m_aNamePlates[ClientID].m_AfkMarkFontSize = FontSize - FontOffsetAfkMark;
+
+				if(m_aNamePlates[ClientID].m_AfkMarkContainerIndex != -1)
+					TextRender()->DeleteTextContainer(m_aNamePlates[ClientID].m_AfkMarkContainerIndex);
+
+				CTextCursor Cursor;
+				TextRender()->SetCursor(&Cursor, 0, 0, FontSize, TEXTFLAG_RENDER);
+				Cursor.m_LineWidth = -1;
+
+				// create nameplates at standard zoom
+				float ScreenX0, ScreenY0, ScreenX1, ScreenY1;
+				Graphics()->GetScreen(&ScreenX0, &ScreenY0, &ScreenX1, &ScreenY1);
+				MapscreenToGroup(m_pClient->m_pCamera->m_Center.x, m_pClient->m_pCamera->m_Center.y, Layers()->GameGroup());
+
+				m_aNamePlates[ClientID].m_AfkMarkWidth = TextRender()->TextWidth(0, FontSize - FontOffsetAfkMark, afkMark, -1);
+
+				m_aNamePlates[ClientID].m_AfkMarkContainerIndex = TextRender()->CreateTextContainer(&Cursor, afkMark);
+				Graphics()->MapScreen(ScreenX0, ScreenY0, ScreenX1, ScreenY1);
+			}
+		}
+
+		ColorRGBA rgbAfkM = color_cast<ColorRGBA>(ColorHSLA(g_Config.m_ClAfkMarkColor));
```

Comment:
```suggestion
		ColorRGBA RgbAfkM = color_cast<ColorRGBA>(ColorHSLA(g_Config.m_ClAfkMarkColor));
```
Proper casing and maybe a better name for it.